script/copy_package.py

```diff
@@ -84,7 +81,7 @@ def bump(v):
     # copy over the package and replace version in manifest and pipeline names
     shutil.copytree(src, dst)
     subprocess.check_call('rm -rf {0}'.format(os.path.join(dst, 'README.template.md')), shell=True)
-    cmd = 'find {0} -not -name "*.png" -type f -print0 | xargs -0 sed -i "" "s/{1}/{2}/g"'.format(
+    cmd = 'find {0} -not -name "*.png" -type f -exec sed -i -e "s/{1}/{2}/g" {{}} \\;'.format(
```

Comment:
I didn't look too deeply into it, but the script wouldn't run for me as it was -- I guess some discrepancy between the version of find shipped with macOS.

Reply:
changes look good - I ran `make update` with the changed code and everything still worked fine on a macOS

Reply:
thanks for testing!